### PR TITLE
Build/environment variables

### PR DESCRIPTION
**Pull request by ccamel**
This PR introduces the parametrization of the HTML `favicon` `title` and `meta tags` (`keywords`, `description`, `version`...) using the ViteJS plugin [vite-plugin-html-config](https://github.com/ahwgs/vite-plugin-html-config). Obvisously, the values are injected in the HTML during build time.

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const config = {
+    apiUrl: "https://example.com/api",
+    appVersion: "1.0.0",
+  };
```
Comment (fredericvilcot) [inline src/config.ts:4]: How do you handle those environment variables?
Will it be part of a next PR that will inject an `env.js` script in the `index.html` file?

File: vite.config.ts [hunk 2]
```diff
@@ -2,13 +2,29 @@ import { defineConfig } from 'vite'
...
     alias: {
       '@': path.resolve(__dirname, './src')
     }
-  }
+  },
+  build: {
+      rollupOptions: {
+        output: {
+          manualChunks: {
+            'react-vendor': ['react', 'react-dom']
```
Comment (fredericvilcot) [inline vite.config.ts:25]: Maybe we could also add `react-router-dom` to this vendor, no?

Comment (ccamel) [reply vite.config.ts]: Ah ah, yes good catch. When I started the PR there was no router yet that's why. I add it of course.

**Review by fredericvilcot — APPROVED** [review]
Great, thanks @ccamel ❤️

**Pull request merged**